pg2avro/pg2avro.py

```diff
@@ -207,6 +205,8 @@ def _get_row_attr(row, attribute: str, fields_schema: Dict):
             None,
         )
         return row[index]
+    elif isinstance(row, object) and type(row) not in BUILTIN_TYPES:
```

Comment:
Could we have a test that would catch this plz?

Reply:
Added test for it. The problem is when the row is some extended class from base type, e.g. OrderedDict (i.e. it is instance of _dict_, but not type _dict_)